.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "discord_lib"]
 	path = discord_lib
-	url = git@github.com:Yarn/discord-lib.git
+	url = https://github.com/Yarn/discord-lib.git
```

Comment:
remove this change, using an http url would prevent me from being able to push changes